src/components/calcite-input/calcite-input.tsx

```diff
@@ -193,9 +241,13 @@ export class CalciteInput {
     const iconScale = this.scale === "s" || this.scale === "m" ? "s" : "m";
 
     const inputClearButton = (
-      <div class="calcite-input-clear-button" onClick={this.clearInputValue}>
+      <button
```

Comment:
Nice update, button is semantically correct here 🧹

Reply:
this also fixed tabbing to the clear button which I think was broken before